content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -79,8 +79,10 @@ After setting these environment variables, run your tests as you normally do:
 NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app DD_CIVISIBILITY_AGENTLESS_ENABLED=true DD_API_KEY=$API_KEY DD_CIVISIBILITY_GIT_UPLOAD_ENABLED=true DD_CIVISIBILITY_ITR_ENABLED=true yarn test
 {{< /code-block >}}
 
+**Important**: by default, `jest` fails if there are no tests to run. It may happen that intelligent test runner skips every test. To prevent `jest` from failing, pass [`--passWithNoTests`][5] to `jest`.
```

Comment:
```suggestion
**Important**:  It is possible for Intelligent Test Runner to skip every test. By default, `jest` fails if there are no tests to run. To prevent `jest` from failing, pass [`--passWithNoTests`][5] to `jest`.
```